docs/ref/files/storage.txt

```diff
@@ -11,7 +11,18 @@ Django provides convenient ways to access the default storage class:
 
 .. data:: storages
 
-    Storage instances as defined by :setting:`STORAGES`.
+    A dictionary-like object that allows you to retrieve a storage instance
+    using its alias as defined by :setting:`STORAGES`.
+
+    ``storages`` has a dictionary attribute ``backends``, which at startup is
+    the raw value provided to :setting:`STORAGES`.
+
+    Additionally ``storages`` provides a ``create_storage()`` method, accepting
+    the dictionary used in :setting:`STORAGES` for the backend, that will
+    return a storage instance. This may be useful for third-party packages
+    needing to instantiate storages in tests::
```

Comment:
Slight rephrasing, and adding the necessary comma after the "Additionally" introductory word:
```suggestion
    Additionally, ``storages`` provides a ``create_storage()`` method that
    accepts the dictionary used in :setting:`STORAGES` for the backend, and
    returns a storage instance. This may be useful for third-party packages
    needing to instantiate storages in tests::
```

Reply:
Yep, fine. 👍